### PR TITLE
Fix permissions for users page

### PR DESCRIPTION
**Pull request by YuriyLu**

### CODE REVIEW

File: app/utils/properties.js [hunk 1]
```diff
@@ -117,6 +102,10 @@ export const getPermissions = profile => {
 };
 
 export const hasGlobalModeratorRole = permissions => {
+  if (!permissions) {
+    permissions =
```
Comment (Lampardks) [inline app/utils/properties.js:106]: don't redefine the incoming variable

File: app/utils/properties.js [hunk 2]
```diff
@@ -117,6 +102,10 @@ export const getPermissions = profile => {
...
+      JSON.parse(getCookie('profileinfols') || '""')?.permissions || [];
+  }
   return !!permissions.find(permission =>
```
Comment (Lampardks) [inline app/utils/properties.js:109]:
```
Boolean(permissions.find(permission =>
  BigNumber.from(permission).eq(DEFAULT_ADMIN_ROLE),
))
```

File: app/utils/properties.js (hunk 1 above)
Comment (YuriyLu) [reply app/utils/properties.js:108]: Done

File: app/utils/properties.js (hunk 2 above)
Comment (YuriyLu) [reply app/utils/properties.js]: Done

**Review by Lampardks — APPROVED** [review]

**Pull request merged**